src/run/runner/wall_time/perf/mod.rs

```diff
@@ -96,10 +97,21 @@ impl PerfRunner {
         };
         debug!("Using call graph mode: {}", cg_mode);
 
+        const PERF_CPU: u32 = 2;
+        const BENCH_CPU: u32 = 3;
+        const BENCH_CPUS: [u32; 2] = [PERF_CPU, BENCH_CPU];
+        const HOST_CPUS: [u32; 2] = [0, 1];
```

Comment:
can you add comments to explain what is going on here ?

especially why we have two host CPUs for example.